kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/IPAddress.java

```diff
@@ -46,20 +46,7 @@
      */
     public static IPAddress getByAddress(byte[] addr) throws UnknownHostException {
         IPAddress result = null;
-        java.net.InetAddress jnetAddr = null;
```

Comment:
@MMaiero I think that removing this would break IPv6 but I agree that this is not an ideal code.